src/platform/tls_schannel.c

```diff
@@ -1656,30 +1677,67 @@ CxPlatTlsIndicateCertificateReceived(
     QUIC_PORTABLE_CERTIFICATE PortableCertificate = {0};
 #endif
 
+    if (PeerCertBlob->Type == QUIC_CERT_BLOB_NONE) {
+        Certificate = NULL;
+        CertificateChain = NULL;
+    } else if (PeerCertBlob->Type == QUIC_CERT_BLOB_CHAIN) {
+#ifndef _KERNEL_MODE
+        CXPLAT_DBG_ASSERT(FALSE);
+        Certificate = NULL;
+        CertificateChain = NULL;
```

Comment:
Why do you have these, when you just set them again in the lines after this?

Reply:
Same below this.

Reply:
I get "uninitialized variable" compilation errors without these. annoying, right?